src/digitalIdentity-frontend/src/app/navigation-bar/navigation-bar.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import {ThemePalette} from '@angular/material/core';
+
+@Component({
+  selector: 'app-navigation-bar',
+  templateUrl: './navigation-bar.component.html',
+  styleUrls: ['./navigation-bar.component.css']
+})
+export class NavigationBarComponent implements OnInit {
+
+  links = ['/', '2', '3', '/DI-Overview', '/register', '/register'];
```

Comment:
what are the links '2' and '3' for?

Reply:
I just use them to distinguish between index 1 and 2 of the list. I think it was to perform some tests